### YamlMime:ManagedReference
items:
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException
  id: EntityDataSourceValidationException
  children:
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  langs:
  - csharp
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException
  type: Class
  summary: "Rappresenta gli errori che si verificano quando la convalida delle proprietà di un&quot;origine dati dinamica."
  syntax:
    content: 'public sealed class EntityDataSourceValidationException : Exception, System.Web.DynamicData.IDynamicValidatorException'
  inheritance:
  - System.Exception
  implements:
  - System.Web.DynamicData.IDynamicValidatorException
  inheritedMembers: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  id: '#ctor'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Crea una nuova istanza di <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> classe con un messaggio fornito dal sistema che descrive l&quot;errore."
  remarks: "Questo costruttore inizializza la <xref:System.Exception.Message%2A>proprietà della nuova istanza di un messaggio fornito dal sistema che descrive l&quot;errore, ad esempio &quot;DefaultMessageDisplayedByParameterlessCtorWriterMustSupply&quot;.</xref:System.Exception.Message%2A> Questo messaggio prende in considerazione le impostazioni cultura correnti.       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un&quot;istanza di <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException>      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| La stringa di messaggio di errore localizzato. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException ();
    parameters: []
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Crea una nuova istanza di <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> classe con un messaggio specificato che descrive l&quot;errore."
  remarks: "Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un&quot;istanza di <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>.</xref:System.Web.UI.WebControls.EntityDataSourceValidationException>      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Stringa del messaggio di errore specificata `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio che descrive l&quot;eccezione. Il chiamante del costruttore è necessario per garantire che la stringa sia stata localizzata per la lingua corrente."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Crea una nuova istanza di <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> classe con un messaggio di errore specificato e un riferimento all&quot;eccezione interna che è la causa dell&quot;eccezione corrente."
  remarks: "Un&quot;eccezione generata come risultato diretto di un&quot;eccezione precedente deve includere un riferimento all&quot;eccezione precedente nella <xref:System.Exception.InnerException%2A>proprietà.</xref:System.Exception.InnerException%2A> Il <xref:System.Exception.InnerException%2A>proprietà restituisce lo stesso valore passato al costruttore o `null` se il <xref:System.Exception.InnerException%2A>proprietà non fornisce il valore di eccezione interna al costruttore.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un&quot;istanza di <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>.</xref:System.Web.UI.WebControls.EntityDataSourceValidationException>      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Stringa del messaggio di errore specificata `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio che descrive l&quot;eccezione. Il chiamante del costruttore è necessario per garantire che la stringa sia stata localizzata per la lingua corrente."
    - id: innerException
      type: System.Exception
      description: "Eccezione che è la causa dell&quot;eccezione corrente. Se il `innerException` parametro non è <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, l&quot;eccezione corrente viene generata in un <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blocco che gestisce l&quot;eccezione interna."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  id: System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  isEii: true
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  type: Property
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Ottiene le eccezioni che si verificano quando un campo dati nuovo o modificato viene convalidato."
  remarks: "Questo membro è un&quot;implementazione esplicita dell&quot;interfaccia di un membro. E può essere utilizzato solo quando il <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>esegue il cast dell&quot;istanza di un <xref:System.Web.DynamicData.IDynamicValidatorException>interfaccia.</xref:System.Web.DynamicData.IDynamicValidatorException> </xref:System.Web.UI.WebControls.EntityDataSourceValidationException>"
  syntax:
    content: System.Collections.Generic.IDictionary<string,Exception> System.Web.DynamicData.IDynamicValidatorException.InnerExceptions { get; }
    return:
      type: System.Collections.Generic.IDictionary{System.String,System.Exception}
      description: "Raccolta che contiene tutte le eccezioni che si sono verificati."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
- uid: System.Collections.Generic.IDictionary{System.String,System.Exception}
  parent: System.Collections.Generic
  isExternal: true
  name: IDictionary<String,Exception>
  nameWithType: IDictionary<String,Exception>
  fullName: System.Collections.Generic.IDictionary<System.String,System.Exception>
  spec.csharp:
  - uid: System.Collections.Generic.IDictionary`2
    name: IDictionary
    nameWithType: IDictionary
    fullName: IDictionary<System.String,System.Exception>
  - name: <
    nameWithType: <
    fullName: <
  - uid: System.String
    name: String
    nameWithType: String
    fullName: String
  - name: ','
    nameWithType: ','
    fullName: ','
  - uid: System.Exception
    name: Exception
    nameWithType: Exception
    fullName: Exception
  - name: '>'
    nameWithType: '>'
    fullName: '>'
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
